Makefile

```diff
@@ -186,9 +179,6 @@ test-slow-integration-report: make-reports-dir get-test-deps test-slow-integrati
 test-slow-integration-report-html: make-reports-dir get-test-deps test-slow-integration
```

Comment:
It makes sense to rename this to `test-intergration-report-html`

Reply:
will do